src/tests/Microsoft.PowerFx.Interpreter.Tests/PadUntypedObjectTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class PadUntypedObjectTests
```

Comment:
naming nit: DataTableUntypedObjectTests? The host it's for initially is kind of irrelevant.

Reply:
True but will have to be redone anyway